### ISSUE TITLE
Unnecessary border on left and right side

### ISSUE THREAD

**Issue by Andreas0602** [jurplel/qView#531]
**Environment:**
- OS: macOS Ventura 13.0.1
- qView version: nightly 2022-11-25.1

**Information:**
When opening a large picture on my iMac with Retina 5K display, qView will show an unnecessary border on the left and right side. See example below.

![example](https://user-images.githubusercontent.com/79788982/206899719-615a2e40-97b3-4c9a-af57-6e5222decdfe.jpg)

**Comment by jurplel** [comment]
I don't think that nightly build has changes merged to master... can you reproduce on any official version?

**Comment by jurplel** [comment]
Should be fixed now from #550, confirm?

**Comment by jurplel** [comment]
Can you list steps to reproduce?

Does this occur using the "window resizes to image size" feature, or is it a general window scaling issue?

**Comment by jurplel** [comment]
That's awfully weird. I wasn't able to reproduce it on my friend's mac. I will try again when i get another chance

**Comment by jdpurcell** [comment]
I am not able to reproduce it either, but I found somewhere in the code that could cause this. If you'd like to test #556 (build artifacts [here](https://github.com/jurplel/qView/actions/runs/3906246964)), @Andreas0602, I'd be curious if the problem is fixed.

**Comment by jdpurcell** [comment]
@Andreas0602 I more thoroughly studied and tested the window sizing/positioning code, and fixed multiple problems which I've described in #556. Only the first problem there should cause unnecessary borders, though, and I had already fixed that yesterday. So I'm not hopeful that my latest changes will fix the border any more than yesterday. Nonetheless, maybe I accidentally fixed something without realizing, or you accidentally ran an older version during part of your testing, so it could be worth trying again with the [latest build](https://github.com/jurplel/qView/actions/runs/3916828003). If that doesn't work I'm really out of ideas.

**Comment by jdpurcell** [comment]
@Andreas0602 Could it be that the maximum size is set very high?
<img width="381" alt="image" src="https://user-images.githubusercontent.com/6741660/212471205-bad395fd-977d-48b4-87e5-48db5329816e.png">

If maximum size were smaller (like 70%?) I wouldn't expect qView to use the full vertical screen area between the menu bar and the dock. I can make the problem happen indeed if I change the maximum to 100%. If that's the cause, I can definitely fix that.

**Comment by jurplel** [comment]
> Indeed, the problem will not occur when I set the maximum size to 70%.
> 
> It would be great if you could fix the problem for the case that the maximum size is set to 100%.

That explains why we were unable to reproduce it--in the future please make sure that you note what settings you are using.

**Comment by jdpurcell** [comment]
@Andreas0602 Side borders due to large maximum size is now fixed in the [latest build](https://github.com/jurplel/qView/actions/runs/3919031841).
